### PR TITLE
T249828: Ignore Error during connection closing

### PR DESCRIPTION
**Pull request by DamianLokic**

### CODE REVIEW

**Comment by DamianLokic** [comment]
I resolved merge conflicts that appeared on previous Pull Request

**Review by viktordick — COMMENTED** [review]
LGTM, but please add a changelog entry

**Comment by DamianLokic** [comment]
I will

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+1.0.9.4 (unreleased)
```
Comment (viktordick) [inline docs/HISTORY.txt]: The version here is wrong, due to something really not obvious: We forked this package and changed a lot of things and have not been able to get them merged upstream (not sure anymore if it is due to a lack of time or due to a lack of interest upstream).
Which means that we must for the time being stay on the version from which we forked (1.0.9.3) and add a specific suffix - i.e. 1.0.9.3+perfact.9

Comment (DamianLokic) [reply docs/HISTORY.txt]: I added another commit which adjusts version and changelog to 1.0.9.3+perfact.9

Comment (DamianLokic) [reply docs/HISTORY.txt]: Do I need to adjust something else for this PR to be merged, so I can create nmu+1 package of perfact-dbutils-zope4 using version of Products.ZPyODBCDA that contains those changes ?

**Review by jan-jockusch — APPROVED** [review]
LGTM

**Comment by DamianLokic** [comment]
@viktordick @jan-jockusch I need this PR to be merged in order to create nmu package of perfact-dbutils-zope4 that overwrites my changes by requiring older version of this package. Could u take a look at this ? Thanks.

**Pull request merged**